src/ProjectNeon/Assets/Data/Enemies/AI/NoAI.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NoAI : TurnAI

```

Comment:
This class can't work. It violates the Liskov Substitution Principle.

Any component in our game is allowed to do a job badly, but he must always do his job.

Reply:
I understand that. 

However, it's inconsistent with the design.

Player take a turn for his whole party, not for each member. And it happens completely asynchronously... it is not triggered like it is for enemies.

Also, the player card selection feature is already finished... so we don't need to plug anything else in for that.